### PR TITLE
Update interface

### PR DESCRIPTION
**Pull request by phantom0212**
Update interface
Edit function

### CODE REVIEW

File: app/Http/Controllers/PersonController.php [hunk 1]
```diff
@@ -23,20 +23,14 @@ public function getAdd(Request $request)
...
         ]);
-        $imageName =
-            $request->file('photo')->getClientOriginalExtension();
-        $name =  $person->id . '_' . $request->file('photo')->getClientOriginalName();
-        $request->file('photo')->move(
-            public_path() . '/upload/images/', $name
-        );
+        $imageName = $request->file('photo')->getClientOriginalExtension();
+        $name = $person->id . '_' . $request->file('photo')->getClientOriginalName();
+        $request->file('photo')->move(public_path() . '/upload/images/', $name);
```
Comment (kakeyang) [inline app/Http/Controllers/PersonController.php]: '/upload/images/' hard coding cases low maintainability. You'd better define it as const.

File: app/Http/Controllers/PersonController.php [hunk 2]
```diff
@@ -23,20 +23,14 @@ public function getAdd(Request $request)
...
         $person->photo = $name;
-        if('name'!=null ||'age'!=null ||'address'!=null||'photo'!=null) {
-
-
+        if ('name' !== null || 'age' !== null || 'address' != null || 'photo' != null) {
```
Comment (kakeyang) [inline app/Http/Controllers/PersonController.php]: Can you check this again? This is really correct?
For example, `'name' !== null` is always true.

**Comment by kakeyang** [comment]
I see you still commit your test data. Please remove them.

File: app/Http/Controllers/PersonController.php [hunk 3]
```diff
@@ -23,20 +23,14 @@ public function getAdd(Request $request)
...
             $person->save();
-
             return redirect('/');
-        }else{
+        } else {
             dd("Something wrong!!!!");
```
Comment (tuananhpham94) [inline app/Http/Controllers/PersonController.php]: I dont think dd is friendly

File: public/app/controllers/PersonController.js [hunk 4]
```diff
@@ -7,10 +7,10 @@ app.controller('PersonController', function ($scope, $http, API) {
         $scope.state = state
         switch (state) {
             case "add" :
-                $scope.frmTitle = "Add new Personal infomation";
+                $scope.frmTitle = "Add new Person";
                 break;
             case "edit" :
-                $scope.frmTitle = "Edit Personal Infomation";
+                $scope.frmTitle = "Edit Infomation";
                 $scope.id = id;
```
Comment (tuananhpham94) [inline public/app/controllers/PersonController.js]: Where can I see this full file

**Pull request merged**